.gitignore

```diff
@@ -1,25 +1,57 @@
+# OS X generated file
+.DS_Store
+
+# built application files
+*.apk
+*.ap_
+
+# files for the dex VM
+*.dex
+
+# Java class files
+*.class
+
 # generated files
+bin/
+gen/
 build/
+build.log
 
 # Local configuration file (sdk path, etc)
 local.properties
-tools/deploy-mvn-artifact.conf
+
+# Eclipse project files
+.settings/
+.classpath
+.project
 
 # Intellij project files
 *.iml
 *.ipr
 *.iws
-.idea/
+/.idea/*
```

Comment:
This seems wrong, having a path that is absolute and anchored in the `/` root directory? In fact we can see in this PR that the whole `.idea/` directory is now commited in git and not ignored anymore after this change. I don't think that's what was expected?

Reply:
Right! This seems suspicious at least, but it looks like it has been there for 2+ years ;-)
Since this PR is a step into bringing the workflow for this repository up to date, I think the right thing to do is to merge this PR into `trunk` as it reflects the current state of the library and, then, open a PR against `develop` for the fix.
Does it make sense to you?

Reply:
Yep, totally, SGTM.